client/app.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import axios from 'axios';
+
+import GameDescription from './components/GameDescription.jsx'
+import Carousel from './components/Carousel.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      gameId: '1',
+      name: null,
+      description: null,
+      videoURL: null,
+      mainImageURL: null,
+      logoURL: null,
+      carouselImagesURL: [],
+      recentReviews: null,
+      allReviews: null,
+      releaseDate: null,
+      developer: null,
+      publisher: null,
+      tags: []
+    }
+    this.fetchGame = this.fetchGame.bind(this);
+  }
+
+  getRandomInt() {
+    return Math.floor(Math.random() * (100)) + 1;
+  }
+
+  fetchGame(gameId) {
+    axios.get('http://localhost:3001/' + gameId)
```

Comment:
You don't want to use a hostname and port here, when you deploy to AWS or where-ever this isn't going to work.

You can simply use ``axios.get(`/${gameId}`)`` instead and the request will use the same hostname and port that the client is already connected on.

Reply:
Fixed